docs/references/dimensions.mdx

```diff
@@ -208,96 +209,7 @@ To help you build your format expression, we recommend using https://customforma
-            format: 'gbp'
-```
-
-These are the options:
-
-| Option  | Equivalent format expression   | Description                                                                         | Raw value  | Displayed value |
-| ------- | ------------------------------ | ----------------------------------------------------------------------------------- | ---------- | --------------- |
-| km      | '#,##0.00" km"'                | Adds the suffix `km` to your value                                                  | 10         | 10 km           |
-| mi      | '#,##0.00" mi"'                | Adds the suffix `mile` to your value                                                | 10         | 10 mi           |
-| usd     | '[$]#,##0.00'                  | Adds the `$` symbol to your number value                                            | 10         | $10.00          |
-| gbp     | '[$£]#,##0.00'                  | Adds the `£` symbol to your number value                                            | 10         | £10.00          |
-| eur     | '[$€]#,##0.00'                  | Adds the `€` symbol to your number value                                            | 10         | €10.00          |
-| jpy     | '[$¥]#,##0.00'                  | Adds the `¥` symbol to your number value                                            | 10         | ¥10             |
-| percent | '#,##0.00%'                    | Adds the `%` symbol and multiplies your value by 100                                | 0.1        | %10             |
-| id      | '0'                            | Removes commas and spaces from number or string types so that they appear like IDs. | 12389572   | 12389572        |
-
-#### Compact (legacy)
-
-You can compact values in your YAML. For example, if I wanted all of my revenue values to be shown in thousands (e.g. `1,500` appears as `1.50K`), then I would write something like this in my .yml:
-
-```yaml
-	version: 2
-	models:
-	- name: sales
-	  columns:
-	    - name: revenue
-	      meta:
-	        dimension:
-	          compact: thousands # You can also use 'K'
-```
-
-| Value     | Alias              | Equivalent format expression            | Example output |
-| --------- | ------------------ | --------------------------------------- | -------------- |
-| thousands | "K" and "thousand" | '#,##0," K"' or '#,##0.00," K"'       | 1K             |
-| millions  | "M" and "million"  | '#,##0,," M"' or '#,##0.00,," M"'     | 1M             |
-| billions  | "B" and "billion"  | '#,##0,,," B"' or '#,##0.00,,," B"'   | 1B             |
-| trillions | "T" and "trillion" | '#,##0,,,," T"' or '#,##0.00,,,," T"' | 1T             |
-
-#### Round (legacy)
-
-You can round values to appear with a certain number of decimal points.
-
-```yaml
-	version: 2
-	models:
-	- name: sales
-	  columns:
-	    - name: revenue
-	      meta:
-	        dimension:
-	          round: 0 # equivalent format expression: '#,##0.0'
-```
-
-</details>
+<FormatExpression/>
```

Comment:
Moved this to a reusable snippet. So it can be edited once for both dimensions and metrics.